src/scripts/core.js

```diff
@@ -1746,6 +1754,46 @@ require([
                                 app.map.infoWindow.show(evt.mapPoint);
                                 calibrationInfoWindow = true;
                             }
+
+                            //Nutrient-Impaired Water Body Site InfoWindow
+                            if (responseObj.layerId === 42) {
+                                var model_ss = "Nutrient-Impaired Water Body";
+                                var auxiliaryTemplate = new esri.InfoTemplate();
+                                auxiliaryTemplate.setTitle(
+                                    model_ss
+                                );
+                                //UPDATE important! make sure the field names below match what is in the REST layer
+                                auxiliaryTemplate.setContent(
+                                    "<div>(2012 303d listing)</div><br>" +
+                                    "<div><b>Unique identifier for impaired water body: </b> " +
+                                        responseObj.feature.attributes
+                                            .WBID +
+                                        "</div><br>" +
+                                        "<div><b>SPARROW Reach ID: </b> </b>" +
+                                        responseObj.feature.attributes
+                                            .Reach_ID +
+                                        "</div><br>" +
+                                        "<div><b>Name of impaired water body: </b>" +
+                                        responseObj.feature.attributes
+                                            .WB_name +
+                                        "</div><br>" +
+                                        "<div><b>State with regulatory authority over the impaired water body: </b>" +
+                                        responseObj.feature.attributes
+                                            .State +
+                                        "</div><br>" 
+                                );
+
+                                var graphic = new Graphic();
+                                var feature = graphic;
+                                responseObj.feature.setInfoTemplate(
+                                    auxiliaryTemplate
+                                );
+                                app.map.infoWindow.setFeatures([
+                                    responseObj.feature,
```

Comment:
Have a look this [http://jsfiddle.net/patelmanya/5SRgH/1/](http://jsfiddle.net/patelmanya/5SRgH/1/) for an example of using the `.resize(width, height)` method on the info window.  This could be set to hard coded values as shown in the example on line 34, or you can get fancy and calculate based off of the title length. as in line 46 of the example.  Since the title is the same length, I think setting a static width is just fine.

Reply:
I think that would be your best bet for getting everything in the title in a single line.

Reply:
Okay, added the resize code and tested it. Not sure how I missed that solution when looking for it the other day. 😐 I think this should be good to go now!